src/common/config/index.ts

```diff
@@ -106,9 +106,17 @@ export class Config extends EventEmitter {
         if (process.platform === 'darwin' || process.platform === 'win32') {
             nativeTheme.on('updated', this.handleUpdateTheme);
         }
-        this.registryConfig = new RegistryConfig();
-        this.registryConfig.once(REGISTRY_READ_EVENT, this.loadRegistry);
-        this.registryConfig.init();
+    }
+
+    initRegistry = () => {
+        return new Promise<void>((resolve) => {
+            this.registryConfig = new RegistryConfig();
```

Comment:
Should we handle calling this function more than one time?

Reply:
I don't think it's necessary, we don't read the registry more than once when the application is started and once it's started I don't believe there's a way for us to find out if the registry has changed and we need to reload the GPO teams.

So I think it's fine as is for now :)